c/compare.sh

```diff
@@ -67,12 +77,23 @@ for f in $SETS ; do
     exit 1
   fi
 
+  # iterate over all files in the set
   i=0
   for ff in $(ls $(grep -v "^#" $f)) ; do
     orig=$ff
 
     if echo $ff | grep -q ".yml$"; then
       input_basename=$(yq --raw-output '.input_files' "$ff")
+      # check whether the input_basename exists (nobody should ever use "null" as a filename!)
+      if [ "null" = "$input_basename" ] ; then
+        echo "No input files defined in $ff" 1>&2
+        exit 1
+      fi
+      # simple check whether input_basename is a list like "[...]"
+      if [ -z "${input_basename##*\[*\]*}" ] ; then
+        echo "ignoring task consisting of multiple sourcefiles"
```

Comment:
Can we make this check a little bit more precise such that only tasks that actually consist of several input files are ignored, and those with a singleton list of input files are handled? Otherwise I think this will be overlooked at some point in the future.